pkgs/development/tools/fdroidserver/default.nix

```diff
@@ -41,12 +41,12 @@ python.pkgs.buildPythonApplication rec {
     paramiko
     pillow
     pyasn1
-    pyasn1-modules
```

Comment:
This is still required https://gitlab.com/fdroid/fdroidserver/-/blob/2.0.3/setup.py#L82